### PR TITLE
Fix config issues

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: kafka_topic_info.go [hunk 1]
```diff
@@ -10,43 +11,89 @@ type KafkaTopicInfo struct {
...
-	parms = setConfInt64(parms, "retention.ms", conf.RetentionMs, -1)
-	parms = setConfInt64(parms, "segment.bytes", conf.SegmentBytes, -1)
-	parms = setConfInt64(parms, "segment.ms", conf.SegmentMs, -1)
+
+	confMods := ConfMods {
+		ConfDeletions: make(map[string]string),
+		ConfAdditions: make(map[string]string)}
+
+	setConf 		(confMods, "cleanup.policy"	,	conf.CleanupPolicyChanged		, conf.CleanupPolicy, "")
+	setConfInt64(confMods, "retention.bytes", conf.RetentionBytesChanged	, conf.RetentionBytes, -1)
```
Comment (dsturnbull) [inline kafka_topic_info.go]: woah nelly

File: kafka_topic_info_test.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      ReplicationFactor: 2,
+      CleanupPolicy: "delete",
+      RetentionBytes: 3,
+      RetentionMs: 4,
+      SegmentBytes: 5,
+      SegmentMs: 6}
+
+  params := topicInfo.alterTopicConfigOpts()
+
+  fmt.Printf("Len: %d\n", len(params))
```
Comment (dsturnbull) [inline kafka_topic_info_test.go]: Couple of debug lines left

Comment (newhoggy) [reply kafka_topic_info_test.go]: I deleted the file 😄

**Comment by dsturnbull** [comment]
That was drastic!

**Comment by newhoggy** [comment]
It was just a useless test file.

**Comment by nkbt** [comment]
Any more insight on what this is, why the change, maybe ticket number?

**Comment by dsturnbull** [comment]
https://mayhem.atlassian.net/browse/MAYH-7894

**Pull request merged**